examples/job/create_and_submit_job.py

```diff
@@ -101,5 +104,7 @@
 
 
 job = job_endpoints.get(job['_id'])
-JSON(job)
-
+if interactive_JSON:
```

Comment:
Nits, but:
(1) `interactive_JSON` => `is_json_viewer_interactive`
(2) let's define it once in the config, create a fn `json_viewer, import it here and use in one line